functions/packages/feed-form/src/impl/feed-form-impl.ts

```diff
@@ -188,3 +176,63 @@ export function buildFeedRow(
     [SheetCol.LogoPermission]: formData.hasLogoPermission,
   };
 }
+
+/**
+ * Sends a Slack webhook message to the configured Slack webhook URL
+ * @param {string} spreadsheetId The ID of the Google Sheet
+ */
+async function sendSlackWebhook(spreadsheetId: string) {
+  const slackWebhookUrl = process.env.SLACK_WEBHOOK_URL;
+  const sheetUrl = `https://docs.google.com/spreadsheets/d/${spreadsheetId}/edit`;
+  if (slackWebhookUrl !== undefined && slackWebhookUrl !== "") {
```

Comment:
Should we expect an error if the Webhook URL is not set instead of silently ignoring the request?

Reply:
From the user's point of view, I don't think they should see an error if we aren't able to get the Slack message. Silently failing in this case is fine, maybe we can add a log for ourselves?

Reply:
Good point; let's add a logging message.